styles.css

```diff
@@ -366,3 +423,458 @@ textarea::-webkit-input-placeholder {
   letter-spacing: 0;
   text-align: left;
 }
+
+.project-list img {
+  transition: 0.7s;
+}
+
+.project-list .laptop:hover {
+  transform: scale(1.02);
+}
+
+@media (min-width: 768px) {
```

Comment:
[OPTIONAL] I recommend you stick to the recommendation from Microverse on media queries, so definitively consider doing this before merging.

> Media Queries for the Desktop breakpoint in the CSS file. (the value for `min-width` property should be in range of `992px` - `1100px`)

Reply:
Ok, I'll do the change, thanks